### ISSUE TITLE
Migrate all databases if multi_db is set

### ISSUE THREAD

**Issue by lynncyrin** [plumdog/django_migration_testcase#36]
I've got an app with two databases

```python
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': 'dayjob',
    },
    'staticpages_db': {
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': 'dayjob_staticpages_db',
    },
}
```

If I want to run migrations against `staticpages_db`, I have to modify `migrate_kwargs` like so

```python
    def migrate_kwargs(self):
        kwargs = super().migrate_kwargs()
        kwargs['database'] = 'staticpages_db'
        return kwargs
```

This is simple to do, but not how I would expect multiple database support to be handled. That expectation being that when django_migration_testcase encounters the `multi_db` setting

```python
class DayJobPrivateRepoTest(MigrationTest):
    multi_db = True
```

It runs the migrations across all available databases. This would mimic the behavior of django's built-in test functions.

I'm more than happy to implement this myself @plumdog! Lemme know what you think

**Comment by plumdog** [comment]
Interesting! I haven't played around with multiple DBs where Django is managing more than one of them.

The simplest case would be where we just want to migrate one database, it's just that it isn't the default one. So this would be a lot like your `kwargs['database'] = 'staticpages_db'` example, just a bit more generic (eg allowing `db = 'staticpages_db'` to be set on the class). I think this would be straightforward to implement, but can currently be achieved without too much hassle.

The slightly fiddlier case would be where we actually want to migrate multiple databases. Eg actually supporting `multi_db = True` and following Django's existing pattern. To handle this the `migrate` method on the `TestCase` would need to run multiple `migrate` commands. Shouldn't be impossible though, but it will be a bit fiddly to write tests for. And this one really does add some new stuff which is do just by overriding methods. This would also cover the simple case above, though if I have many databases, maybe I want to be able to select which ones to migrate so the test isn't too slow? We can revisit this bit eventually if needed though.

-------------

TLDR: yup, this sounds totally reasonable and definitely adds useful things. Supporting `multi_db = True` would be ace.

**Comment by lynncyrin** [comment]
I'm closing a bunch of my older github issues ♻️  feel free to open a new issue describing this same change! Or re-open this one, if someone is still interested in working on it 📝
